co.udea.api.hero.exception.DataNotFoundException#instance-init#